app/src/main/java/ch/epfl/skysync/database/FirestoreDatabase.kt

```diff
@@ -91,6 +91,7 @@ class FirestoreDatabase(private val useEmulator: Boolean = false) {
   ): T? {
     val documentSnapshot = db.collection(path).document(id).get().await()
 
+    println(clazz.java)
```

Comment:
You shouldn't have a print there. If you want to show some informations you should use the Log class

Reply:
Fixed. This was for testing purposes and we do not need a log here.